lib/main.js

```diff
@@ -25,7 +25,11 @@ const main = module.exports = (searchString, options, callback) => { // eslint-d
   const currentRef = options.nextpageRef;
   // Setting cookie in request headers to get safe search results
   const requestOptions = options.safeSearch ? { headers: { Cookie: 'PREF=f2=8000000' } } : {};
-  UTIL.getPage(currentRef ? UTIL.buildFromNextpage(currentRef) : UTIL.buildLink(searchString), requestOptions, (err, body) => { // eslint-disable-line consistent-return, max-len
+  // Geo related parameters
+  // Setting default values
+  if (!options.gl) options.gl = 'US';
+  if (!options.hl) options.hl = 'en';
```

Comment:
atm we're setting default parameters twice, both here and in the definition of the buildlLink/buildFromNextpage functions - depending on how we solve the problem in Line 118-122 we might get rid of one

Reply:
You're right, but I thought this way the code looks more descriptive.

While parameters/fields are having their default values set, a contributor (like me) can see all possible field names.

Default values in buildLink/buildFromNextpage is for safety, because those parameters are required, otherwise the function will return incorrect result. This is done for future, if the functions will be used outside of current scope.